package.json

```diff
@@ -3,13 +3,14 @@
   "version": "0.20.1",
   "description": "UI Toolkit",
   "license": "MIT",
-  "main": "index.js",
+  "main": "dist/index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/holidayextras/ui-toolkit.git"
   },
   "scripts": {
     "build": "scripts/build.sh",
+    "postbuild": "ulimit -n 9999; grunt build",
```

Comment:
grunt build calls some shell commands anyway, I'd like to get rid of grunt and put it all in build.sh but that's probably for another pr